### PR TITLE
updating tutorial files

### PR DESCRIPTION
**Pull request by thisisjofrank**

### CODE REVIEW

File: runtime/tutorials/read_write_files.md [hunk 1]
```diff
@@ -1,109 +0,0 @@
----
-title: "Read and Write Files"
-oldUrl:
-  - /runtime/manual/examples/read_write_files/
```
[the remark this answers is not in the document]
Comment (thisisjofrank) [reply runtime/tutorials/read_write_files.md:4]: Yes, we definitely need that.

File: runtime/tutorials/connecting_to_databases.md [hunk 2]
```diff
@@ -26,14 +32,20 @@ const client = await new Client().connect({
...
 
 ### Connect to Postgres with deno-postgres
 
-```ts
+First, import the `Client` class from the `deno-postgres` module and create a
+new client instance. Then connect to the database passing an object with the
+connection details:
+
+```ts class="main.js"
 import { Client } from "https://deno.land/x/postgres/mod.ts";
```
[the remark this answers is not in the document]
Comment (thisisjofrank) [reply runtime/tutorials/connecting_to_databases.md:49]: not sure I understand the question.

[the remark this answers is not in the document]
Comment (marvinhagemeister) [reply runtime/tutorials/connecting_to_databases.md:49]: We have a section `Connect to Postgres with deno-postgres` that references this module. Later we have a section `Connect to Postgres with postgresjs` which references the same module. I'm mainly wondering if they're supposed to refer to different modules.

[the remark this answers is not in the document]
Comment (thisisjofrank) [reply runtime/tutorials/connecting_to_databases.md:49]: they're two different modules tho
https://deno.land/x/postgres
https://deno.land/x/postgresjs

File: runtime/tutorials/read_write_files.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (thisisjofrank) [reply runtime/tutorials/read_write_files.md:4]: have added a new redirects file called 'oldurls' let me know if there could be a better name for this.

File: oldurls.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "/runtime/manual/examples/read_write_files/": "/examples/",
+  "/runtime/tutorials/unix_cat/": "/examples/unix-cat/",
+  "/examples/dependency-management/": "runtime/fundamentals/configuration/#dependencies"
```
Comment (marvinhagemeister) [inline oldurls.json:4]:
```suggestion
 "/examples/dependency-management/": "/runtime/fundamentals/configuration/#dependencies"
```

**Pull request merged**